src/Sylius/Component/Core/Test/Services/DefaultUnitedStatesChannelFactory.php

```diff
@@ -206,14 +199,16 @@ private function createCountry()
     /**
      * @return CurrencyInterface
      */
-    private function provideCurrency()
+    private function provideCurrency($currencyCode = null)
     {
+        $currencyCode = $currencyCode ?: self::DEFAULT_CURRENCY_CODE;
```

Comment:
I think we can always use `USD` in this case, it's "The United States" channel, so the currency is pretty much known and immutable :) In Default channel it makes sense, but for this specific factory it doesn't IMO.